platform/admin/ui/src/main/webapp/js/views/module/ExportModal.view.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+/* global define */
+define([
+        'icanhaz',
+        'underscore',
+        'marionette',
+        'backbone',
+        'jquery',
+        'js/views/Modal',
+        'text!templates/module/ExportModal.handlebars',
+        'spin',
+        'spinnerConfig',
+        'js/models/Alerts.js',
+        'js/views/Alerts.view',
+        'js/models/module/Export.js'
+    ],
+    function (ich, _, Marionette, Backbone, $, Modal, exportModal, Spinner, spinnerConfig, AlertsModel, AlertsView, Export) {
+
+        ich.addTemplate('exportModal', exportModal);
+
+        var ExportModal = Modal.extend({
+            template: 'exportModal',
+            model: new Export(),
+            regions: {
+                jolokiaError: '.alerts'
+            },
+            modelEvents: {
+                'change:warnings': function () {
+                    this.updateAlerts('warnings');
```

Comment:
Seems like the view should be listening for model events instead of the model calling a method on the view.  The model should not know anything about the view.